### PR TITLE
OO-762 adding the right endpoints for the mailing and the mail templates

### PR DESCRIPTION
**Pull request by mcmpp**

### CODE REVIEW

**Comment by fabianpage** [comment]
Maybe pull out text, ids and url as a parameter? (I'm not sure if that is easy)

**Comment by AlwinEgger** [comment]
I agree. Let's make the core directive independent from the use case.

File: app/scripts/common/components/oo-email-dialog.controller.js [hunk 1]
```diff
@@ -5,46 +5,64 @@
...
       $scope.batchCreate = function() {
-        $scope.kundenToMailTo.ids = $scope.kundeIds;
-        MailerService.sendEMailToKunden($scope.kundenToMailTo).then(function() {
-          $scope.commandIssued = true;
-          $scope.createHasWorked = true;
-        }, function (){
-          $scope.commandIssued = true;
-          $scope.createHasWorked = false;
-        });
+          console.log("Aqui estoy!!")
```
Comment (AlwinEgger) [inline app/scripts/common/components/oo-email-dialog.controller.js]: Drop this

File: app/scripts/common/components/oo-email-dialog.controller.js [hunk 2]
```diff
@@ -5,46 +5,64 @@
...
+          console.log($scope)
+        if ($scope.ids){
+          console.log("Ya estoy dentro!!")
```
Comment (AlwinEgger) [inline app/scripts/common/components/oo-email-dialog.controller.js]: ...

**Review by AlwinEgger — APPROVED** [review]
Don't forget to bump the version and set it in openolitor-client-admin

**Pull request merged**